server/pulp/server/async/worker_watcher.py

```diff
@@ -17,97 +17,43 @@
 from gettext import gettext as _
 import logging
 
-from pulp.common import constants
 from pulp.server.async.tasks import _delete_worker
-from pulp.server.db.model import Worker, ResourceManagerLock
+from pulp.server.db.model import Worker
 
 
 _logger = logging.getLogger(__name__)
 
 
-def _parse_and_log_event(event):
+def handle_worker_heartbeat(worker_name):
```

Comment:
Can a log statement be added here to log at DEBUG level with the name of the worker? This would be like the log statement in [Issue 2516](https://pulp.plan.io/issues/2516). It probably should also include the `datetime.utcnow()` timestamp that is being written to the database even though the logs already probably contain timestamps.